main.go

```diff
@@ -79,6 +81,8 @@ func main() {
 
 		fmt.Print("\033[u")
 
+		// Disable cursor
+		fmt.Print("\033[?25l")
```

Comment:
Something in the loop seems to reset this so you have to print it on every iteration instead of just once at the top :man_shrugging:

Reply:
Changed how tty logic is handled so this no longer applies